src/components/Pages/Members/Members.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { getFakeUsers } from '../../../mockApi/getData';
+import { getMemberItems } from '../../../shared/helpers';
+import { PageTitle } from '../../PageTitle/PageTitle';
+import { TABLE_TITLES, TITLES_PAGES, BUTTONS_NAMES } from '../../../shared/constants';
+import { Table } from '../../Table/Table';
+import style from './Members.module.css';
+
+export class Members extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+    };
+  }
+
+  async componentDidMount() {
+    const users = await getFakeUsers();
+    const items = await getMemberItems(users);
```

Comment:
move `const users = await getFakeUsers();` in method

Reply:
```suggestion
  async componentDidMount() {
    const members = await getFakeUsers();
    this.setState({ members });
  }
```